src/main/java/com/todolist/app/model/ToDoListAppConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.todolist.app.model;
+
+
+public final class ToDoListAppConstants {
+    /**
+     * This constant represents add button on see-all-tasks page.
+     */
+    private static final String ADD_OPERATION = "ADD";
+
+    private ToDoListAppConstants() {
+    }
+
+    public static String getAddOperation(){
```

Comment:
Sorry it was probably my mistake about constants class. All the constants there should be `public static final` there is no need in additional get methods.